lib/galaxy/tools/actions/history_imp_exp.py

```diff
@@ -30,7 +30,7 @@ def execute(self, tool, trans, incoming={}, set_output_hid=False, overwrite=True
         job.tool_id = tool.id
         job.user_id = trans.user.id
         start_job_state = job.state  # should be job.states.NEW
-        job.state = job.states.WAITING  # we need to set job state to something other than NEW, or else when tracking jobs in db it will be picked up before we have added input / output parameters
+        job.state = job.states.NOT_READY    # we need to set job state to something other than NEW, or else when tracking jobs in db it will be picked up before we have added input / output parameters
```

Comment:
```suggestion
        job.state = job.states.NOT_READY  # we need to set job state to something other than NEW, or else when tracking jobs in db it will be picked up before we have added input / output parameters
```